### PR TITLE
Snackbars misc fixes

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,6 @@
     "vite-plugin-vuetify": "^2.0.3",
     "vue": "^3.3.11",
     "vue-router": "^4.4.0",
-    "vuetify": "^3.6.10"
+    "vuetify": "^3.6.13"
```
Comment (rjuju) [inline package.json:39]: is that actually needed? if not I'd rather not bump it, or bump as many things as possible in a single giant commit in a different pr. but please don't bump each sub components one at a time.

Comment (pgiraud) [reply package.json]: OK. I removed this commit. This doesn't fix an error, only a warning.

Comment (rjuju) [reply package.json]: ok. we could probably plan to bump the various dependencies a bit before releases.

**Review by rjuju — APPROVED** [review]

**Comment by pgiraud** [comment]
I don't have the permissions to merge.

**Comment by rjuju** [comment]
really? you're supposed to have a commit bit on at least this repo

**Comment by pgiraud** [comment]
Screenshot from 2024-07-29 09-19-56

**Comment by rjuju** [comment]
that's weird, maybe additional permissions are needed. I will look at it shortly (I'm not in front of a computer right now)

**Comment by pgiraud** [comment]
All good. Thanks.

**Comment by rjuju** [comment]
Great!

**Pull request merged**